{"name": "Проверка ошибки при авторизации курьера с несуществующими данными", "status": "passed", "description": "Отправляем запрос на авторизацию в сервисе с несуществующими данными и проверяем ответ", "start": 1711982911298, "stop": 1711982911777, "uuid": "7dca6cc4-8191-44bc-82b8-bcf9901ce05e", "historyId": "7d6c240669be102bb96f7f568dd3ce34", "testCaseId": "7d6c240669be102bb96f7f568dd3ce34", "fullName": "tests.test_login_courier.TestLoginCourier#test_courier_login_without_null_login_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "WIN-B72I5GSJN4S"}, {"name": "thread", "value": "15276-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}